### ISSUE TITLE
How do I use the responsive mixins?

### ISSUE THREAD

**Issue by anointed** [anthonyshort/stitch-css#17]
I've imported stitch into my project but when I try the following I just get errors:
.responsive-nav-markup {
  display: none;
  @include mobile-landscape {
    display: block;
  }
}

When I compile the Error says: Syntax error: Invalid CSS after "...bile-landscape ": expected "}", was "{"

*If it matters, here is my compass/sass info:
C:\Users\admin>sass --version
Sass 3.2.0.alpha.94 (Bleeding Edge)

C:\Users\admin>gem install stitch
Fetching: stitch-0.1.3.2.gem (100%)
Successfully installed stitch-0.1.3.2
1 gem installed
Installing ri documentation for stitch-0.1.3.2...
Installing RDoc documentation for stitch-0.1.3.2...

C:\Users\admin>gem env
RubyGems Environment:
- RUBYGEMS VERSION: 1.8.16
- RUBY VERSION: 1.9.3 (2012-02-16 patchlevel 125) [i386-mingw32]
- INSTALLATION DIRECTORY: C:/Ruby193/lib/ruby/gems/1.9.1
- RUBY EXECUTABLE: C:/Ruby193/bin/ruby.exe
- EXECUTABLE DIRECTORY: C:/Ruby193/bin
- RUBYGEMS PLATFORMS:
  - ruby
  - x86-mingw32
- GEM PATHS:
  - C:/Ruby193/lib/ruby/gems/1.9.1
  - C:/Users/shawn/.gem/ruby/1.9.1
- GEM CONFIGURATION:
  - :update_sources => true
  - :verbose => true
  - :benchmark => false
  - :backtrace => false
  - :bulk_threshold => 1000
- REMOTE SOURCES:
  - http://rubygems.org/

**Comment by anointed** [comment]
I also noticed that when i simply include the stitch I end up with the following on the top of my compiled css file. not sure why.

/\* line 4, ../../sass/stitch/patterns/images/_image-replace.scss */
.__image-replace {
  display: block;
  text-indent: -9999em;
  overflow: hidden;
  background-repeat: no-repeat;
  text-align: left;
  direction: ltr;
}

/\* 
    Add an icon to an inline element. E.g anchor or span
    Pass through an asset, and optionally a padding value, x-position and y-position.

```
.icon {
    @include inline-icon('icons/gowalla_16.png', 30px, right);
}

<a href="#" class="icon">Sweet icon</a>
```

_/
/_ line 7, ../../sass/stitch/patterns/layout/_clear-floats.scss _/
.__clearfix {
  zoom: 1;
}
/_ line 9, ../../sass/stitch/patterns/layout/_clear-floats.scss _/
.__clearfix:before, .__clearfix:after {
  content: "";
  display: table;
}
/_ line 13, ../../sass/stitch/patterns/layout/_clear-floats.scss */
.__clearfix:after {
  clear: both;
}

/\* line 5, ../../sass/stitch/patterns/legacy/_inline-block.scss */
.__inline-block {
  display: inline-block;
  *display: inline;
  *zoom: 1;
}

**Comment by chrisbuttery** [comment]
Hi anointed,

To utilise these responsive mixing you'll also need to import the stitch patterns.
So where your importing compass and stitch…

@import 'compass';
@import 'stitch';
@import 'stitch/patterns/responsive';

 0.1.4 will be released very shortly and will include these patterns by default (an fix the comment issue).

**Comment by anthonyshort** [comment]
You're actually using the wrong version of Stitch. You'll need to install the prerelease version 

gem install stitch --pre

Stitch uses classes for common mixins, like the clearfix, to lower the amount of code that is generated. If we didn't take this approach there would be the same clearfix properties applied everywhere.

**Comment by anthonyshort** [comment]
But also what Chris said.

**Comment by anointed** [comment]
Hi guys n thanks for the info
Here is my latest attempt.

I did the gem install stitch --pre it installed 0.1.4beta2

I then add this to my scss file
@import "compass";
@import 'stitch';
@import 'stitch/patterns/responsive';

/\* Main Navigation
..................................*/
.responsive-nav-markup {
  display: none;
  @include mobile-landscape {
display: block;
}
}

error I receive when compass.app tries to compile the css says:

Syntax error: Invalid CSS after "...bile-landscape ": expected "}", was "{"
        on line 9 of C:/websites/compass.dev/assets/sass/partials/main-navigation.scss

Am I missing a step somewhere?
*sorry brand new to scss/compass/compass.app/stitch but loving what I am finding so far

Maybe it is a problem with http://compass.handlino.com/ ?

**Comment by anthonyshort** [comment]
Ah. Yes I think Compass app can't load Compass extensions. I'm not entirely sure because I haven't used that app much. Try and do it from terminal/command line and see if it works. I'll need to take a look at making it work with Compass.app if that's the case

**Comment by anointed** [comment]
Thnx, sent you a contact message to gmail from address in your profile. Hope it helps.

**Comment by anointed** [comment]
Just wanted to report back that I got it working with the compass.app program.

Here is what I did.

Click on preferences in compass.app
then under compass version, I chose custom and used the path below: (windows)
C:\Ruby193\lib\ruby\gems\1.9.1\ as the path.
*that is where all the gems seem to download when I install a gem, and I did check and both the stitch and stitch beta folders are both in there.

Now the css stylesheet is compiling properly.

Hope this info helps someone out there
